RSDKv5/RSDK/Graphics/EGL/EGLRenderDevice.cpp

```diff
@@ -283,7 +283,7 @@ bool RenderDevice::InitGraphicsAPI()
     videoSettings.fsWidth  = 1920;
     videoSettings.fsHeight = 1080;
 #elif RETRO_PLATFORM == RETRO_ANDROID
-    customSettings.maxPixWidth = 510;
```

Comment:
This is more or less added to reflect v3v4 behavior. Need @Rubberduckycooly's opinion on this one.

Reply:
Mania will basically self destruct above 512 (it already becomes highly unstable above 424 anyways), but in theory there's nothing wrong on the engine side of things so for now we can remove the limit and take it as it comes